src/UrlBarSynch.jsx

```diff
@@ -0,0 +1,62 @@
+const queryString = require('query-string');
```

Comment:
It could be a `js` file I suppose, but I was also just reading about React containers that are renderless components, so, :shruggie: